configuration/rules-blockly-standard-ext.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: documentation
+title: Rules Blockly - openHAB Extensions to the Standard
+---
+
+# openHAB Extensions to the Standard
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+## Introduction
+
+This section explains only the blocks that have been added to the standard blocks by openHAB
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+{: #blockly-standard-extension-overview}
+
+## Colors
+
+The Colors section simplifies the selection of  Color Values. Technically these color RGB values are provided as a hexadecimal combination of the three values for red, green and blue (hence RGB). Each of the individual colors can range from 0 - 255 or 0-FF (hexadecimal). A color of r=255, g=255, b=255 would therefore result into #ffffff preceded by a # which identifies it as a hexadecimal number.
+
+![colors-overview](images/blockly/blockly-colors-overview.png)
+
+Note that typically RGB values are not used within openHAB for color states but instead the Color-Type expects a HSB Type. See the special hsb-conversion block below.
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.